src/Editor.elm

```diff
@@ -22,6 +22,7 @@ import Tableau exposing (..)
 import Task
 import UndoList exposing (UndoList)
 import Validate
+import Validation.Common
```

Comment:
Rename `Validate.elm` to `Validation.elm` so that the whole validation code looks like a single “package”. Also, it seems saner to
```elm
import Validation.Common exposing (Problem(..), ProblemType(..))
```
everywhere `Validation.Common` is imported and remove the qualifier `Validation.Common.` from type signatures.